src/Command/DecodeCommand.php

```diff
@@ -34,10 +37,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $data = file_get_contents($data);
         }
 
-        // TODO : add a progress bar
+        try {
+            file_put_contents(
+                __DIR__ . '/../../files/decode/output.json',
```

Comment:
I had to `json_encode` the result again to print it, this is quite bad. I couldn't use the first json because the pretty_print option takes too much space in the console.